lib/matplotlib/tri/triplot.py

```diff
@@ -42,6 +44,7 @@ def triplot(ax, *args, **kwargs):
     linestyle, marker, color = matplotlib.axes._base._process_plot_format(fmt)
 
     # Insert plot format string into a copy of kwargs (kwargs values prevail).
+    kwargs = cbook.normalize_kwargs(kwargs, mlines.Line2D)
     kw = kwargs.copy()
```

Comment:
Just a small comment: As `cbook.normalize_kwargs` already creates a new dictionary, this additional copy might be superfluous and thus not necessary anymore.

Reply:
Thanks. Made the Required Changes. Please Review.

Reply:
Thanks.
Made the required changes. 
Please Review.